service/src/main/java/org/sunbird/actor/otp/OTPActor.java

```diff
@@ -240,4 +254,185 @@ private String getType(String type) {
         return null;
     }
   }
+
+
+  /*
+   * This method generates OTP for the provided request.
+   */
+  private void generateOTPV3(Request request) {
+    logger.debug(request.getRequestContext(), "OTPActor:generateOTP method call start.");
+    // Extract necessary parameters from the request.
+    String type = (String) request.getRequest().get(JsonKey.TYPE);
+    String key = (String) request.getRequest().get(JsonKey.KEY);
+    String userId = (String) request.getRequest().get(JsonKey.USER_ID);
+    String contextType = (String) request.getRequest().get(JsonKey.CONTEXT_TYPE);
+    List<String> contextAttributesList = (new ObjectMapper()).convertValue(request.get(JsonKey.CONTEXT_ATTRIBUTES),
+            new TypeReference<>() {
+            });
+    String contextAttributes = String.join(",", contextAttributesList);
```

Comment:
Null check is required

Reply:
1. Null check is implemented.
2. If it is a null value it is handled as empty string.